lib/sourceParams.js

```diff
@@ -137,6 +137,16 @@ module.exports = function(app) {
   app.set('cssCompiledOutput', path.normalize(appDir + params.cssCompiledOutput));
   app.set('jsCompiledOutput', path.normalize(appDir + params.jsCompiledOutput));
 
+  // keep track of state of env
+  if (process.env.NODE_ENV === 'development') {
+    // set the envStateDev to true if in dev mode
+    app.set('envStateDev', true);
```

Comment:
we don't need to track this in an app var. querying `process.env.NODE_ENV` each time this is needed is fine